web/admin/application/configs/klear/PlatformPortalsList.yaml

```diff
@@ -0,0 +1,123 @@
+#include conf.d/mapperList.yaml
+#include conf.d/actions.yaml
+#include conf.d/documentationLink.yaml
+
+production:
+  main:
+    module: klearMatrix
+    defaultScreen: platformPortalList_screen
+  screens: &platformPortal_screensLink
+    platformPortalList_screen: &platformPortalList_screenLink
+      controller: list
+      rawCondition: "WebPortal.brand IS NULL"
+      forcedValues:
+        urlType: god
+      pagination:
+        items: 25
+      <<: *WebPortals
+      class: ui-silk-world-link
+      title: _("List of %s %2s", ngettext('Platform Portal', 'Platform Portals', 0), "[format| (%parent%)]")
+      info:
+        <<: *documentationLink
+        href: "/doc/en/administration_portal/platform/brands.html#brand-urls"
```

Comment:
Just in case, this URL still works, or this is a broken link now?

Reply:
It was broken, thanks :)